ocs_ci/ocs/resources/pod.py

```diff
@@ -1032,8 +1032,14 @@ def get_pvc_name(pod_obj):
         pod_obj (str): The pod object
 
     Returns:
-        pvc_name (str): The pvc_name on a given pod_obj
+        pvc_name (str): The pvc_name on a given pod_obj,
```

Comment:
there's no such variable pvc_name - please remove it.
Also, under 'Returns':
str: The pvc_name on a given pod_obj ....